tools/loris_db_upgrade.php

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/php
+<?php
+
+/**
+ * This script will upgrade the loris database to the lastest. 
+ * Usage: php loris_db_upgrade.php [-D]
```

Comment:
```suggestion
 * Usage: php loris_db_upgrade.php
```

The `-D` flag doesn't seem to be used in the script.